tools/lib/optimizeGlb.js

```diff
@@ -54,4 +53,40 @@ function optimizeGlb(glbBuffer, options) {
                     return getBinaryGltf(gltf, embed, embedImage).glb;
                 });
         });
-}
\ No newline at end of file
+}
+
+function fixBatchIdSemantic(gltf) {
+    var meshes = gltf.meshes;
+    for (var meshId in meshes) {
+        if (meshes.hasOwnProperty(meshId)) {
+            var primitives = meshes[meshId].primitives;
+            var primitivesLength = primitives.length;
+            for (var i = 0; i < primitivesLength; ++i) {
+                var attributes = primitives[i].attributes;
+                for (var semantic in attributes) {
+                    if (attributes.hasOwnProperty(semantic)) {
+                        if (semantic === 'BATCHID') {
```

Comment:
Is it a bad idea to directly look up `attributes.BATCHID`? If not, it might be a little cleaner.